server/controllers/events/events_controller_e2e_test.go

```diff
@@ -959,6 +959,10 @@ func setupE2E(t *testing.T, repoDir string) (events_controllers.VCSEventsControl
 		AggregateApplyRequirements: &events.AggregateApplyRequirements{
 			WorkingDir: workingDir,
 		},
+		PullStatusFetcher: boltdb,
+		StaleCommandChecker: &events.StaleCommandHandler{
+			Counter: statsScope.SubScope("stale_checker").Counter("dropped_commands"),
```

Comment:
hmm not sure this flows well with our current metrics hierarchy. we basically have:
```
production:app:atlantis
     /cmd
        /autoplan
        /comment
           /plan
           /apply
     /scheduled
```

I like to keep things as intuitive as possible so its easy to search and also not necessarily name them after the owning struct since those can change but your metrics shouldn't.

How about something like this: `production:app:atlantis:cmd:stale`?